opentelemetry-exporter-google_cloud_trace/CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Contributing to OpenTelemetry operations Ruby
+
+1. **Sign one of the contributor license agreements below.**
+2. Fork the repo, develop and test your code changes.
+3. Send a pull request.
+
+## Contributor License Agreements
+
+Before we can accept your pull requests you'll need to sign a Contributor
+License Agreement (CLA):
+
+- **If you are an individual writing original source code** and **you own the
+  intellectual property**, then you'll need to sign an [individual
+  CLA](https://developers.google.com/open-source/cla/individual).
+- **If you work for a company that wants to allow you to contribute your work**,
+  then you'll need to sign a [corporate
+  CLA](https://developers.google.com/open-source/cla/corporate).
+
+You can sign these electronically (just scroll to the bottom). After that, we'll
+be able to accept your pull requests.
+
+## Setup
+
+This is a mono repo that will contain tools that will support OpenTelemetry instrumentation for Google Cloud using Ruby.
+
+1. Install Ruby. OpenTelemetry operations Ruby requires Ruby 2.7+. You may choose to
+   manage your Ruby and gem installations with [RVM](https://rvm.io/),
+   [rbenv](https://github.com/rbenv/rbenv), or
+   [chruby](https://github.com/postmodern/chruby), or
+   [asdf](https://github.com/asdf-vm/asdf).
+
+2. Install [Bundler](http://bundler.io/).
+
+   ```sh
+   $ gem install bundler
+   ```
+
+3. Install the top-level project dependencies.
+
+   ```sh
+   $ bundle install
+   ```
+
+4. Install the specific tool dependencies.
+
+   ```sh
+   $ cd opentelemetry-exporter-google_cloud_trace/
+   $ bundle install
+   ```
+
+## GCP trace exporter tests
+
+Tests are very important part of OpenTelemetry operations Ruby. All contributions
+should include tests that ensure the contributed code behaves as expected.
+
+To run the unit tests, documentation tests, and code style checks together for a
+package:
+
+``` sh
+$ cd opentelemetry-exporter-google_cloud_trace/
+$ bundle exec rake ci
```

Comment:
I thought we didn't wanna continue using Rake and instead make use of toys.

Reply:
We use toys in our CI(Github actions and kokoro runs), there is rake task too in case contributors need to run it locally with rake.

Reply:
Yeah, I meant for that rake task. That could potentially be configured via toys too?

Reply:
I felt we can leave it here as such for the ease of contributors coming with experience of using Rake already. Also it is the same way in our other repo, would leave it for consistency too. 

Probably can we discuss this in a separate issue as its out of scope of this PR which tries to fix documentation generation ?

Reply:
Let's discuss it offline.